### PR TITLE
added gems and follow required version tools

### PR DESCRIPTION
**Pull request by TheEyePatch**
This PR is for installing gems that are already inside the Gemfile and tools that matches the required versions (ruby 2.7.2, nodejs 12.18.3 and yarn 1.22.4) as given in the .tool-versions file.

### CODE REVIEW

File: .tool-versions [hunk 1]
```diff
@@ -1,3 +1,4 @@
-ruby 2.7.2
-yarn 1.22.4
+Ruby 2.7.2
+Rails 6.0.3.4
 nodejs 12.18.3
+yarn 1.22.4
```
Comment (EliezerAmbion) [inline .tool-versions:4]: undo this change

Comment (EliezerAmbion) [inline .tool-versions]: undo this change

**Review by EliezerAmbion — CHANGES_REQUESTED** [review]

File: .tool-versions [hunk 2]
```diff
@@ -1,3 +1,4 @@
 ruby 2.7.2
-yarn 1.22.4
+rails 6.0.3.4
 nodejs 12.18.3
+yarn 1.22.4
```
Comment (TheEyePatch) [inline .tool-versions]: changed first letters of tools to small case

File: .tool-versions [hunk 3]
```diff
@@ -1,3 +1,4 @@
 ruby 2.7.2
-yarn 1.22.4
+rails 6.0.3.4
```
Comment (analyn-cajocson) [inline .tool-versions]: Remove `rails 6.0.3.4` here

File: .tool-versions (hunk 2 above)
Comment (analyn-cajocson) [inline .tool-versions]: Add new line at the end of file

**Review by analyn-cajocson — CHANGES_REQUESTED** [review]

File: .tool-versions [hunk 4]
```diff
@@ -1,3 +1,3 @@
 ruby 2.7.2
 nodejs 12.18.3
-yarn 1.22.4
\ No newline at end of file
+yarn 1.22.4
```
Comment (TheEyePatch) [inline .tool-versions:3]: Removed rails in .tool-version and added new line

**Review by analyn-cajocson — APPROVED** [review]

**Review by EliezerAmbion — APPROVED** [review]
